index.js

```diff
@@ -0,0 +1,150 @@
+// ЗАДАЧИ логические операторы
+
+// Проверка значения из диапазона
+
+if (age >= 14 && age <= 90);
+
+// Проверка значения вне диапазона
+if (!(age >= 14 && age <= 90));
+
+if (age < 14 || age > 90);
```

Comment:
why do you need an empty if? add {} and do for example console.log here

we don't write like this never

Reply:
fixed